docs/source/blocks/block-style-wrapper.md

```diff
@@ -0,0 +1,99 @@
+  "property=og:title": "Blocks - Style Wrapper"
+  "keywords": "Volto, Plone, frontend, React, Blocks, Edit, Style, Wrapper, components"
+---
+
+# Blocks - Style Wrapper
+
+The block style wrapper is part of a block anatomy.
+It allows you to inject styles from the block schema into the block wrapper in the form of class names.
+It wraps the block edit and the view components.
+
+It can be used mapping directly unique values to CSS properties, like `background-color` mapped to a single color.
+However, the real intention is to bring the ability to create a way to define complex CSS and a full set of styles mapped to a CSS class.
+For example, very often, applying a background color is not enough, and you need to modify also the font color as well, in order to make it readable because of contrast issues.
+The style object field can be extended to hold any number of fields, depending on your needs.
+The resultant set of class names to be injected are built concatenating the key-value pairs of the `style` object field, separated by `--` and prefixing the result with the `has--` prefix.
+The class name generator algorithm supports up to one level of nested objects, constructing the concatenated CSS class from the nested structure as well.
+See below for an example.
+
+## Enabling Style Wrapper in a block
+
+The wrapper is always present in the render of the view and the edit components.
+If you want to add the default set of styles, you need to enable with the following flag:
+
+```js
+  // (in your block config object)
+  my_custom_block: {
+    // (more block settings)
+    stylesEnabled: true,
+  }
+```
+
+```{note}
+This will work if your block uses `BlocksForm` component to define an schema driven block configuration settings.
+```
+
+This will add a new fieldset `Styling` to your block schema settings.
+By default, only a `backgroundColor` property is set, configured by: `defaultSchema` in `src/components/manage/Blocks/Block/StylesSchema.jsx`.
+
+## Extending the default styling field
+
+You can modify the default set of styles, by using a `schemaEnhancer` function, in the same way that you would do it for any block schema enhancer.
+Use the `stylesSchema` key in your block config object as follows:
+
+```js
+import { styleSchemaEnhancer } from '@plone/volto/components/manage/Blocks/Block/StylesSchema';
+
+  // (in your block config object)
+  my_custom_block: {
+    // (more block settings)
+    stylesEnabled: true,
+    stylesSchema: myCustomStyleSchema
+  }
+```
+
+```{note}
+The signature for a `schemaEnhancer` is `({schema, formData, intl})`.
+```
+
+## The `styles` field
+
+The `style` field is mapped to an `objectWidget`, and the `stylesSchema` adds the fields into this field, creating an object that is the sum of all of the widgets assigned to it, and its values.
```

Comment:
The use of `style` here does not align with the preceding heading's use of `styles`. Is it singular or plural? Please make sure they match.

```suggestion
The `style` field is mapped to an `objectWidget`.
The `stylesSchema` adds the fields into this field, creating an object that is the sum of all of the widgets assigned to it and its values.
```

Reply:
The heading uses `styles` (plural), and the paragraph uses `style` (singular). Please use one or the other for both.